packages/0x.js/package.json

```diff
@@ -33,7 +33,7 @@
         "upload_docs_json": "aws s3 cp generated_docs/index.json $S3_URL --profile 0xproject --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --content-type application/json"
     },
     "config": {
-        "artifacts": "TokenTransferProxy Exchange TokenRegistry Token EtherToken",
+        "artifacts": "Exchange DummyToken ZRXToken Token WETH9 TokenTransferProxy MultiSigWallet MultiSigWalletWithTimeLock MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress MaliciousToken TokenRegistry Arbitrage EtherDelta AccountLevels",
```

Comment:
Why?

Reply:
So here is the issue:

We need a canonical 0x protocol smart contract migration. This migration will be used to:
1. Deploy 0x protocol on new blockchains (mainnt, testnets, cosmos?)
2. To setup Ganache with the 0x smart contracts for testing purposes (0x.js tests need this)

Before this PR, the migrations were in `contracts` which satisfied 1, but not 2. Since we now use in-process ganache, we cannot simply call the migrations via an NPM script. We need access to the migrations source code.

Maybe the only viable solution here is to have the migrations live in their own package. Deployments to blockchains can happen from that package, and it can also be included into 0x.js. What do you think?

Reply:
Ended up moving migrations into separate package.